src/enkf/gridio_fv3reg.f90

```diff
@@ -591,6 +591,7 @@ subroutine writegriddata(nanal1,nanal2,vars3d,vars2d,n3d,n2d,levels,ndim,vargrid
     tsen_ind  = getindex(vars3d, 'tsen')  ! Tv (3D)
     q_ind   = getindex(vars3d, 'q')   ! Q (3D)
     cw_ind  = getindex(vars3d, 'cw')  ! CWM for WRF-NMM
+    oz_ind  = getindex(vars3d, 'oz')  ! Oz (3D)
```

Comment:
Does this PR add line 594 oz_ind?

Reply:
Yes,  it was found this line was missing in the current GSI enkf and need to be fixed, though this is in fact not related to the current issue (parallel IO ).

Reply:
This line is supposed to tell the code if  oz is a control variable.